### ISSUE TITLE
[PMM-13141] Expose `featureCompatibilityVersion`

### ISSUE THREAD

**Issue by tumd** [percona/mongodb_exporter#740]
**Is your feature request related to a problem? Please describe.**
1. Before upgrading to a new major version of a node/replica you must verify that all nodes have correct featureCompatibilityVersion set on all nodes.
It would be convenient to only have to glance a Grafana dashboard to assert this is correctly set on all nodes before upgrade.
2. It would be nice to setup an alert that triggers if "featureCompatibilityVersion" and `mongodb_version_info{mongodb="x.y.z"}` differ, notifying you that you have forgotten to update this after a major version upgrade.

**Describe the solution you'd like**
Either add a new metric which expose a label with the value of `featureCompatibilityVersion`, or add this label to the already existing `mongodb_version_info` metric.

```js
db.adminCommand( { getParameter: 1, featureCompatibilityVersion: 1 } )
```

**Describe alternatives you've considered**
None at the moment.

**Additional context**
None at the moment.

**Comment by tumd** [comment]
This seem to have been fixed in #863, and released with 0.41.0 - however completely undocumented ([REFERENCE.md](https://github.com/percona/mongodb_exporter/blob/main/REFERENCE.md), [CHANGELOG](https://github.com/percona/mongodb_exporter/blob/main/CHANGELOG) & release notes).

**Comment by BupycHuk** [comment]
Good point, will update documentation
